src/main.rs

```diff
@@ -1,7 +1,94 @@
 // SPDX-License-Identifier: Apache-2.0
 
-//! `sevctl` is a tool for managing the AMD Secure Encrypted Virtualization (SEV) Platform
-//! Secure Processor (PSP).
+//! # sevctl
+//!
+//! `sevctl` is a command line utility for managing the AMD Secure Encrypted Virtualization (SEV) platform.
+//! It currently supports the entire management API for the Naples generation of processors.
+//!
+//! ## Usage
+//!
+//! ### help
+//!
+//! Every `sevctl` (sub)command comes with a quick `--help` option for a reference on its use. For example:
+//!
+//! ```
```

Comment:
oh, this should be 
````suggestion
//! ```console
````

for every command line example

Reply:
Good catch